engine/shaderc/src/test/data/bindings.vert

```diff
@@ -0,0 +1,31 @@
+#version 140
+
+in vec4 position;
+in vec3 normal;
+in vec2 tex_coord;
+
+uniform matrices
+{
+    mat4 mtx_world;
+    mat4 mtx_view;
+    mat4 mtx_projection;
+    mat4 mtx_normal;
+};
+
+uniform extra
+{
+    vec4 offset;
+} extra_variable;
```

Comment:
Part of the reason we need this library, there is no way to get the instance variable name here (extra_variable) from the client tool! But now we can :)